project2/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+
+
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles.css">
+
+
+    <title>Aesop will love this,,,, for sure</title>
+</head>
+
+<body>
+    <section class="container">
+
+        <img src="images/aesop_logo.png" alt="logo of Aesop" class="logo">
```

Comment:
Note that the width of your logo is constrained to 157px, but the file is almost 800px wide.
Similarly, your image is over 1000px wide, but the card is never larger than 548px. These images should both be resized and optimized:
<img width="981" alt="Screen Shot 2019-10-28 at 9 04 42 AM" src="https://user-images.githubusercontent.com/2653980/67680668-0eaf6e00-f962-11e9-8155-bfb26acc5231.png">